bayesian/utils.py

```diff
@@ -340,15 +340,21 @@ def build_nested_schema_dict(schema_dict):
     return result
 
 
-def get_next_component_from_graph(ecosystem, user_id):
+def get_next_component_from_graph(ecosystem, user_id, company):
     tag_count = 2
-    qstring = ("user = g.V().has('userid', '{uid}').next();"
+    qstring = ("user = g.V().has('userid','{uid}').tryNext().orElseGet{{graph.addVertex("
+               "'vertex_label', 'User', 'userid', '{uid}', {company})}};"
```

Comment:
"'vertex_label', 'User', 'userid', '{uid}', {company})}};"
Extra comma after '{uid}' needs to be removed